estimagic/dashboard/monitoring_callbacks.py

```diff
@@ -149,6 +151,7 @@ def _update_monitoring_tab(
         param_cds (bokeh.ColumnDataSource)
 
     """
+    clip_bound = 1e40
```

Comment:
Please move this to `estimagic.config`

Reply:
I implemented the solution proposed by @hmgaudecker (0afeae5).